test/end2end_test.go

```diff
@@ -888,6 +888,23 @@ type lazyConn struct {
 	beLazy int32
 }
 
+// errMsgContainsPossibleStopMessages addresses around the timing of when TCP packets
```

Comment:
This comment should be attached to the function.

The consts can be defined inside the function, too.

Also please take a look at this comment again.  The first bit reads a little awkwardly to me right now.